seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -220,6 +242,11 @@
       "descriptionEs": [
         "Compara y actualiza el inventario actual, almacenado en un arreglo bidimensional, contra otro arreglo bidimensional de inventario nuevo. Actualiza las cantidades en el inventario actual y, en caso de recibir una nueva mercancía, añade su nombre y la cantidad recibida al arreglo del inventario en orden alfabético.",
         "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/FreeCodeCamp-Get-Help' target='_blank'>Read-Search-Ask</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
+      ],
+      "titleIt": "Aggiornamento dell'inventario",
+      "descriptionIt": [
+        "Confronta e aggiorna l'inventario, contenuto in un array bidimensionale, con un secondo array bidimensionale contenente una nuova consegna. Aggiorna le quantità disponibili in inventario (dentro <code>arr1</code>). Se uno degli articoli non è presente nell'inventario, aggiungi allo stesso il nuovo articolo e la sua quantità. Ritorna un array con l'inventario aggiornato, che deve essere ordinato alfabeticamente per articolo.",
```

Comment:
Try to avoid repetition "...**contenuto**...**contenente**..." , e.g. replacing ~~contenente~~ with **relativo ad** or something similar.